### PR TITLE
feat: Npm cache comments for issue bot

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #1, Fixes #2` 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/14669"></a>

### CODE REVIEW

**Comment by liuxingbaoyu** [comment]
I didn't modify the actual content of the other comments, just that the formatting plugin automatically removed useless spaces.

**Comment by liuxingbaoyu** [comment]
I didn't modify the actual content of the other comments, just that the formatting plugin automatically removed useless spaces.

File: .github/workflows/issue-triage.yml [hunk 1]
```diff
@@ -81,4 +81,31 @@ jobs:
...
+
+            1. Please check if there is a proxy or cache between you and npm
+            (this is the most likely cause and it is common in corporate networks).
+
+            2. Visit https://status.npmjs.org/ to see if the current npm is working properly.
+
+            3. Open the package's home page on npm like https://www.npmjs.com/package/@babel/core
+            and check if the version you want exists.
+
+            4. See https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67
```
Comment (nicolo-ribaudo) [inline .github/workflows/issue-triage.yml:111]:
```suggestion
            This seems to be an npm related issue: you can verify that the package is available using the npm website.
            For example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types`
            (replace `@babel/types` in the URL with the package you want to check). If you see zero downloads don't worry:
            npm can take up to a few days to display the downloads count, but this does not mean that it's not working.
            
            Although we can't handle it, we provide some common workarounds for your reference.

            1. Please check if there is a stale proxy or cache between you and npm
            (this is the most likely cause and it is common in corporate networks).

            2. Visit https://status.npmjs.org/ to see if the current npm is working properly.

            3. See https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67
```

**Review by nicolo-ribaudo — COMMENTED** [review]
@JLHwung unfortunately we all know that there are still many people that completely ignore the link in the issue template :sweat_smile:

File: .github/workflows/issue-triage.yml (hunk 1 above)
Comment (liuxingbaoyu) [reply .github/workflows/issue-triage.yml]: Oops, I remembered to state the number of downloads, but I forgot it.😂

File: .github/workflows/issue-triage.yml [hunk 2]
```diff
@@ -81,4 +81,32 @@ jobs:
...
+        if: |
+          github.event.action == 'labeled' &&
+          github.event.label.name == 'i: npm cache'
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          issue: ${{ github.event.issue.number }}
+          comment: >
+            Hi @${{ github.event.issue.user.login }}!
+
+            This seems to be an npm related issue: you can verify that the package is available using the npm website.
```
Comment (existentialism) [inline .github/workflows/issue-triage.yml:100]: > an npm related issue

Wonder if we should call it an "unavailable package issue" instead?

File: .github/workflows/issue-triage.yml [hunk 3]
```diff
@@ -81,4 +81,32 @@ jobs:
...
+            For example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types`
+            (replace `@babel/types` in the URL with the package you want to check). If you see zero downloads don't worry:
+            npm can take up to a few days to display the downloads count, but this does not mean that it's not working.
```
Comment (existentialism) [inline .github/workflows/issue-triage.yml:103]:
```suggestion
            npm can take up to a few days to display the downloads count, but this does not mean that it's not available.
```

File: .github/workflows/issue-triage.yml [hunk 4]
```diff
@@ -81,4 +81,32 @@ jobs:
...
+            
+            Although we can't handle it, we provide some common workarounds for your reference.
```
Comment (existentialism) [inline .github/workflows/issue-triage.yml:105]: _Technically_ none of the following are really workarounds, howabout:

```suggestion
            Here are some things you can try:
```

File: .github/workflows/issue-triage.yml (hunk 2 above)
Comment (nicolo-ribaudo) [reply .github/workflows/issue-triage.yml:100]:
```suggestion
            This seems to be an issue caused by a package that you are not able to download: you can verify that the package is available using the npm website.
```

File: .github/workflows/issue-triage.yml [hunk 5]
```diff
@@ -81,4 +81,32 @@ jobs:
...
+          github.event.action == 'labeled' &&
+          github.event.label.name == 'i: npm cache'
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          issue: ${{ github.event.issue.number }}
+          comment: >
+            Hi @${{ github.event.issue.user.login }}!
+
+            This seems to be an issue caused by a package that you are not able to download: you can verify that the package is available using the npm website.
+            For example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types`
```
Comment (nicolo-ribaudo) [inline .github/workflows/issue-triage.yml:101]:
```suggestion
            This seems to be an issue caused by a package that you are not able to download.
            You can verify that the package is available using the npm website:
            for example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types`
```

**Pull request merged**